### DISCUSSION TITLE
iphone use writeFile(…)

### DISCUSSION THREAD

**Discussion by todayandyesterday** [ionic-team/capacitor#4287] [Q&A]
hi:
  I wrote the picture in Base64 to Documents, but I can't find the picture written in Album。
 I hope God can answer it。thanks！
![1614825397(1)](https://user-images.githubusercontent.com/70627671/109902600-a307ae80-7cd5-11eb-9e98-c1d4fd473994.png)

**Comment by jcesarmobile** [comment]
Filesystem plugin on iOS writes inside the app sandbox, which is only accessible from the app itself. The album is a shared “storage”, but filesystem plugin can’t write there.
There are 3rd party plugins for saving pictures in the album.

**Reply by todayandyesterday** [reply]
Hello, can't you show it on the Album if it's not on the Album

**Reply by jcesarmobile** [reply]
No, to show it in the album you have to save it to the album with some plugin.

**Reply by todayandyesterday** [reply]
‘red party plugins’ ，I can't find it，Can you tell me where it's linked on GitHub，thanks！

**Reply by jcesarmobile** [reply]
It was the auto complete, I meant 3rd party. Means there are plugins for saving in the gallery. Not sure if any capacitor plugin, but there are a lot of Cordova plugins and they work on capacitor.
